Public/Src/FrontEnd/Nuget/NugetSpecGenerator.cs

```diff
@@ -115,68 +119,80 @@ public ISourceFile CreatePackageConfig()
+                        {
+                            compile.Add(CreateSimpleBinary(assembly));
+                        }
                     }
-                }
 
-                // Runtime items
-                if (TryGetValueForFrameworkAndFallbacks(analyzedPackage.Libraries, new NugetTargetFramework(framework.Key), out IReadOnlyList<RelativePath> libAssemblies))
-                {
-                    foreach (var assembly in libAssemblies)
+                    // Runtime items
+                    if (TryGetValueForFrameworkAndFallbacks(analyzedPackage.Libraries, new NugetTargetFramework(monikers.First()), out IReadOnlyList<RelativePath> libAssemblies))
                     {
-                        runtime.Add(CreateSimpleBinary(assembly));
+                        foreach (var assembly in libAssemblies)
+                        {
+                            runtime.Add(CreateSimpleBinary(assembly));
+                        }
                     }
-                }
 
-                // Dependency items
-                if (analyzedPackage.DependenciesPerFramework.TryGetValue(
-                    framework.Key,
-                    out IReadOnlyList<INugetPackage> dependencySpecificFrameworks))
-                {
-                    foreach (var dependencySpecificFramework in dependencySpecificFrameworks)
+                    // Dependency items
+                    if (analyzedPackage.DependenciesPerFramework.TryGetValue(monikers.First(), out IReadOnlyList<INugetPackage> dependencySpecificFrameworks))
                     {
-                        dependencies.Add(CreateImportFromForDependency(dependencySpecificFramework));
+                        foreach (var dependencySpecificFramework in dependencySpecificFrameworks)
+                        {
+                            dependencies.Add(CreateImportFromForDependency(dependencySpecificFramework));
+                        }
                     }
-                }
 
-                dependencies.AddRange(analyzedPackage.Dependencies.Select(CreateImportFromForDependency));
-
-                cases.Add(
-                    new CaseClause(
-                        new LiteralExpression(framework.Key.ToString(m_pathTable.StringTable)),
-                        new ReturnStatement(
-                            new CallExpression(
-                                PropertyAccess("Managed", "Factory", "createNugetPackge"),
-                                new LiteralExpression(analyzedPackage.Id),
-                                new LiteralExpression(analyzedPackage.Version),
-                                PropertyAccess("Contents", "all"),
-                                Array(compile),
-                                Array(runtime),
-                                Array(dependencies)
+                    cases.Add(
+                        new CaseClause(
+                            new LiteralExpression(monikers.Last().ToString(m_pathTable.StringTable)),

```

Comment:
>Last [](start = 59, length = 4)

It might be nicer to use 'first' here and when adding the other cases jsut do `.Skip(1)`